include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -294,4 +310,12 @@
 #    define _ONEDPL_SYCL_INTEL_COMPILER 1
 #endif
 
+// Intel(R) oneAPI DPC++ Compiler cannot build some cases of destroy and destroy_n with unseq policies on Windows
+// It might be fixed in the future releases, which follow 20240000
+#if defined(_MSC_VER) && __INTEL_LLVM_COMPILER <= 20240000
+#    define _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDWOS_BROKEN 1
+#else
+#    define _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDWOS_BROKEN 0
+#endif
```

Comment:
typo, need to correct its usage too.

```suggestion
#if defined(_MSC_VER) && __INTEL_LLVM_COMPILER <= 20240000
#    define _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN 1
#else
#    define _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN 0
#endif
```

Reply:
Unresolving until the usage is also fixed (so it isn't forgotten).

Reply:
Fixed